filebeat/tests/system/test_multiline.py

```diff
@@ -48,11 +50,13 @@ def test_c_style_log(self):
             path=os.path.abspath(self.working_dir) + "/log/*",
             multiline=True,
             pattern="\\\\$",
-            match="after"
+            match="before"
```

Comment:
it used to pass, cause number of lines was same. but when testing for C-like endings, the ending must be special on the line before.

The validation only checks the number of events, but not the line contents.